packages/examples/nextjs-starter/pages/_app.tsx

```diff
@@ -10,7 +10,7 @@ require("@solana/wallet-adapter-react-ui/styles.css");
 const App: FC<AppProps> = ({ Component, pageProps }) => {
   return (
     <SolanaAuthProvider
-      domain="desolate.space"
+      domain="example.xyz"
```

Comment:
What do you think of naming this authDomain for consistency with the server side env var ?

Reply:
Yeah! In nextJS, any environment variable whose name begins with "NEXT_PUBLIC_" is accessible both in the browser and server

https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser

Reply:
Also remember to change the property name here, this woul not compile